sky/utils/resources_utils.py

```diff
@@ -0,0 +1,63 @@
+"""Resources related utilities."""
```

Comment:
We can call this module accelerator_registry? So it reminds people if we want to add a new accelerator, this is the module to change (see a related comment below).

Reply:
Great point. Fixed!